app/views/topics/show.html.erb

```diff
@@ -8,6 +8,7 @@
     <% end  %>
   </div>
   <div class="col-md-4">
-    <h1>link to delete</h1>
+    <%= link_to "Edit Topic", edit_topic_path, class: "btn btn-block btn-lg btn-warning" %>
```

Comment:
Does `edit_topic_path` generate the expected path? It looks like this is missing information.